.devcontainer/optimized/.devcontainer.json

```diff
@@ -0,0 +1,11 @@
+// The docker image used below was generated from '.devcontainer/devcontainer.json'
+// by the '.github/workflows/dev-container.yml' workflow.
+//
+// By building this dev container image in advance, tools 
+// like GitHub Codespaces (https://containers.dev/supporting) 
+// do not need to install all the prerequsite dependencies from scratch, 
+// getting you into your development environment faster!
+{
+    "name": "SerenityOS (Pre-Built Image)",
+    "image": "ghcr.io/SerenityOS/serenity:base"
+}
```

Comment:
Whoops, missed this one: another missing newline on EOF

Reply:
Ah, and when this one gets fixed up, the fixup commit(s) should be fixup-rebased into the commit that introduced the change, rather than added as a separate commit. (which appears to be the second commit for all of them?)